repository/c.json

```diff
@@ -415,6 +415,16 @@
 				}
 			]
 		},
+		{
+			"name": "Catppuccin", 
```

Comment:
Typically it's a good idea to add "color scheme", or in this case "colors schemes" in the name.